components/molecules/ItemHeaderDropdown.vue

```diff
@@ -0,0 +1,48 @@
+<template>
```

Comment:
I would rename `ItemHeaderDropdown` to `HeaderDropdownItem`. It matches better our naming convention.